### PR TITLE
Switch ru and uk lemmatizers to pymorphy3

### PR DESCRIPTION
**Pull request by adrianeboyd**
## Description

Switch ru and uk lemmatizers to pymorphy3

### Types of change

?

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by d9nchik** [comment]
Sorry for the late answer, but I would like to mention the key benefits of using **pymorphy3** over **pymorphy2**
## TLDR: Main changes between pymorphy2 and pymorphy3
- Enlarge the size of suffixes support [`unsigned short` to `unsigned int`](https://github.com/no-plagiarism/pymorphy3/commit/6de75017a10b2ebea78e33b110f5bc166a139d81)
- Recompile [pymorphy-dict-uk from the largest Ukrainian dictionary](https://github.com/brown-uk/dict_uk)
## Now, let's dive deeper
### Prehistory
The main idea has been to refresh `pymorphy2` dictionaries, mainly _Ukrainian_, that were in betta.
When we've tried to recompile the _Ukrainian_ dictionary error with a **small suffix size** has been encountered.

Further exploration has shown, that changing this variable makes dictionaries incompatible with each other. (breaking **backwards** compatible).
### Release
We've tried to make a [contribution to the project](https://github.com/dchaplinsky/LT2OpenCorpora/pull/26). We've even spoken with the main contributors, but he disappeared and the need for the library improvement staidly.
I've written simple instructions for my friends.
```md
1. install original pymorphy2 `pip install pymorphy2`
2. go to venv/lib/pymorphy2/dawg.py
3. Change in 46 line 
`    DATA_FORMAT = str(">HHH")` to
`    DATA_FORMAT = str(">IHH")`
4. Save
5. Copy in venv/lib new pymorphy2_dicts_ru and pymorphy2_dicts_uk
6. happy hacking!

How to create new dictionaries.
1. clone pymorphy2-dicts
2. install all dependencies from requirements.txt
3. Do steps 2-4 from the previous instruction
4. Compile wanted dictionary
5. You have a new dictionary
```
This wasn't helpful for **developers**, especially for **DevOps** staff(building containers, testing code in CI and so on), so we've decided to [release library `pymorphy3` with updated dictionaries](https://github.com/no-plagiarism/pymorphy3).

### Summary
I hope, this brief article has shown our aims and our path. Also, we actively hoping to rise up: we're looking for a repo maintainer.  Even a small contribution can make the whole world better.
